packages/expo-notifications/android/src/main/java/expo/modules/notifications/service/NotificationForwarderActivity.kt

```diff
@@ -15,6 +16,14 @@ class NotificationForwarderActivity : Activity() {
     super.onCreate(savedInstanceState)
     val broadcastIntent =
       NotificationsService.createNotificationResponseBroadcastIntent(applicationContext, intent.extras)
+
+    val foregroundLaunchActivityIntent = ExpoHandlingDelegate.getForegroundLaunchActivityIntent(this)
+    if (foregroundLaunchActivityIntent != null) {
+      val notificationResponse = NotificationsService.getNotificationResponseFromBroadcastIntent(broadcastIntent)
+      NotificationsService.setNotificationResponseToIntent(foregroundLaunchActivityIntent, notificationResponse)
+      startActivity(foregroundLaunchActivityIntent)
```

Comment:
I see this piece of code is also present in `ExpoHandlingDelegate` itself: https://github.com/expo/expo/blob/660574385842ff87d0811be4b9968b8693317038/packages/expo-notifications/android/src/main/java/expo/modules/notifications/service/delegates/ExpoHandlingDelegate.kt#L133-L138

Maybe we should unify these paths? From what I can see `NotificationForwarderActivity` is only a proxy for broadcasting/delegating heavy stuff to some other classes, so I guess it should not duplicate the code from `ExpoHandlingDelegate`